static/src/javascripts/projects/common/modules/identity/modules/modal.js

```diff
@@ -4,6 +4,35 @@ import fastdom from 'lib/fastdom-promise';
 const ERR_MODAL_NOT_FOUND = 'Modal not found';
 const ERR_MODAL_MALFORMED = 'Modal is malformed';
 
+const removeNewsletterReminders = (modalEl: HTMLElement) => {
+    const newsletterReminders = modalEl.querySelector(
+        `.identity-consent-journey-modal-reminder`
+    );
+    if (newsletterReminders != null) {
+        while (newsletterReminders.firstChild) {
```

Comment:
As a general rule, all dom operations need to be wrapped inside `fastdom`, but especially writes.